common/helpers/campShortTitle.js

```diff
@@ -0,0 +1,59 @@
+const AUTOMATIC_SHORTNERS = [
+  { regex: /So(?:mmer)?la(?:ger)?/gi, text: 'SoLa' },
+  { regex: /Pfi(?:ngst)?la(?:ger)?/gi, text: 'PfiLa' },
+  { regex: /He(?:rbst)?la(?:ger)?/gi, text: 'HeLa' },
+]
+
+const ADDITIONAL_SHORTNERS = [
+  { regex: /Auslandssola/gi, text: 'AuLa' },
+  { regex: /Auffahrtslager/gi, text: 'AufLa' },
+  { regex: /Abteilungslager/gi, text: 'AbLa' },
+  { regex: /Klassenlager/gi, text: 'KlaLa' },
+  {
+    regex: /(Bez)irks(So|Ab|Au|Auf|He|Pfi)La|(Ka)ntonslager|(Bu)ndeslager/gi,
+    text: '$1$2$3$4La',
+  },
+  {
+    regex:
+      /(?<start>.*?\b)?(?<camp>[a-z]{2,}La(?:ger)?)(?<middle>(?:\s(?!19|20))|.*?)\s?(?:(?:19|20)(?<year>\d{2}))(?<end>.*)/gi,
+    text: '$<start>$<camp>$<year>$<middle>$<end>',
+  },
+  { regex: /(Pfadi|Pio|Rover)stufe(?!n)/g, text: '$1s' },
+  { regex: /(B)ieberstufe(?!n)/g, text: '$1ieberli' },
```

Comment:
```suggestion
  { regex: /(Bie?)berstufe(?!n)/g, text: '$1berli' },
```
In german, it is `Biber`.